tests/behat/features/field.feature

```diff
@@ -1,4 +1,4 @@
-@d7
+@d7 @d8
```

Comment:
here and below - making tests for Drupal version-agnostic traits to run for d7 and d8 builds